pkg/config/duration_test.go

```diff
@@ -0,0 +1,76 @@
+package config
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDuration_MarshalJSON(t *testing.T) {
+	tests := []struct {
+		name  string
+		input Duration
+		want  string
+	}{
+		{"zero", *MustNewDuration(0), `"0s"`},
+		{"one second", *MustNewDuration(time.Second), `"1s"`},
+		{"one minute", *MustNewDuration(time.Minute), `"1m0s"`},
+		{"one hour", *MustNewDuration(time.Hour), `"1h0m0s"`},
+		{"one hour thirty minutes", *MustNewDuration(time.Hour + 30*time.Minute), `"1h30m0s"`},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			b, err := json.Marshal(test.input)
+			assert.NoError(t, err)
+			assert.Equal(t, test.want, string(b))
+		})
+	}
+}
+
+func TestDuration_Scan_Value(t *testing.T) {
+	t.Parallel()
+
+	d := MustNewDuration(100)
+	require.NotNil(t, d)
+
+	d.Duration()
```

Comment:
nit:  I think this might be a stray line

Reply:
Fixed